python_modules/libraries/dagster-duckdb-pandas/dagster_duckdb_pandas/duckdb_pandas_type_handler.py

```diff
@@ -48,8 +48,8 @@ def handle_output(self, context: OutputContext, table_slice: TableSlice, obj: pd
                 "dataframe_columns": MetadataValue.table_schema(
                     TableSchema(
                         columns=[
-                            TableColumn(name=name, type=str(dtype))
-                            for name, dtype in obj.dtypes.iteritems()
+                            TableColumn(name=str(name), type=str(dtype))
```

Comment:
is this change intentional?

Reply:
Thanks @benpankow. `itertimes` is deprecated since pandas version 1.5.0, hence I changed it to `items`.
However, I'm thinking now that it might result in issue for users with an older version of pandas. I can revert it for now but keeping a note to change it in future?

Reply:
hey @pzarabadip i'm going to go ahead and revert this change so we can merge in the PR - I'll make a note of it so i can address it in a follow up pr for cleaning up this integration

Reply:
hey @jamiedemaria Sure and thanks.